src/value.rs

```diff
@@ -147,11 +155,94 @@ impl Number {
     }
 }
 
+impl Number {
+    /// Construct a new number.
+    pub fn new(v: impl Into<Number>) -> Self {
+        v.into()
+    }
+
+    /// Returns the `f64` representation of the number regardless of whether the number is stored
+    /// as a float or integer.
+    /// 
+    /// # Example
+    ///
+    /// ```
+    /// # use ron::value::Number;
+    /// let i = Number::new(5);
+    /// let f = Number::new(2.0);
+    /// assert_eq!(i.get(), 5.0);
+    /// assert_eq!(f.get(), 2.0);
+    /// ```
+    pub fn get(self) -> f64 {
```

Comment:
should be called `into_f64`

Reply:
Should we have a deprectaed `get` that is an alias for `into_f64` for backwards compatibility?

Reply:
I thought your PR is API breaking anyway, isn't it?

Reply:
You're right. sorry, since floats are strictly not equal to integers of the same value. Do you think it's worthwhile to try to preserve this equality at all for compatibility or otherwise?